resources/views/content/top-links.blade.php

```diff
@@ -1,41 +1,39 @@
-<x-section class="mb-3 flex flex-col gap-2">
-    <a class="btn text-center py-2" href="/globalRanking.php?s=5&t=2">
-        <span class="text-yellow-400"><x-fas-medal/></span>
+<x-section class="top-links">
+    <a href="/globalRanking.php?s=5&t=2">
+        <span class="text-yellow-500"><x-fas-medal/></span>
         Global Ranking
     </a>
-    <a class="btn text-center py-2" href="{{ route('download.index') }}">
-        <span class="text-heading"><x-fas-gamepad/></span>
+    <a href="{{ route('download.index') }}">
+        <span class="text-purple-400"><x-fas-gamepad/></span>
```

Comment:
The blue color was more on-brand than this pinkish hue.

Reply:
Blue was just the theme-based color being applied though. I think one reason I've changed it is that the Discord icon (not shown on ss) is also blue.

Reply:
The Discord and Patreon icons use their respective brand color. I'd prefer the other icons to stay in the theme color.
The only other icon to have a different color than the default theme color should be the ranking as that's a bit of an exception. Other resources like games do not have any color assigned.